app/wizard_pipelines/feedback/steps/rating.rb

```diff
@@ -0,0 +1,24 @@
+module Feedback
+  module Steps
+    class Rating < ::GITWizard::Step
+      RATINGS = [
+        OpenStruct.new(id: 0, value: "Very satisfied"),
+        OpenStruct.new(id: 1, value: "Somewhat satisfied"),
+        OpenStruct.new(id: 2, value: "Neither satisfied nor dissatisfied"),
+        OpenStruct.new(id: 3, value: "Somewhat dissatisfied"),
```

Comment:
@StackedPancakez We are recording this as `Dissatisfied in the database so we will essentially be mis-recording what users select

I've checked the GDS manual and they use `Dissatisfied` rather than `Somewhat dissatisfied` so please could you change the front end label to `Dissatisfied` (and keep the data capture in the database the same)